src/local/butler/common.py

```diff
@@ -145,7 +145,10 @@ def _print(s):
     if print_output:
       print(s)
 
-  _print('Running: %s' % command)
+  print_string = 'Running: %s' % command
+  if cwd:
+    print_string += " (dir='%s')" % cwd
```

Comment:
super nit: maybe `(cwd='%s')` ? `dir` is a bit ambiguous

Reply:
Done!